modules/gittar/helper/git_cmd.go

```diff
@@ -201,7 +202,16 @@ func RunProcess(service string, c *webcontext.Context) {
 
 		repository := c.MustGet("repository").(*gitmodule.Repository)
 		if preReceiveHook(pushEvents, c) {
-
+			// Only when one branch is created will it be written to the writer
+			// Refer to github
+			if len(pushEvents) == 1 && pushEvents[0].IsCreateNewBranch() {
+				// TODO write create pipeline link to writer
+				//c.GetWriter().Write(NewReportStatus(
+				//	"unpack ok",
+				//	"ok "+pushEvents[0].Ref,
+				//	fmt.Sprintf("\nCreate a pipeline request for '%s' on Erda by visiting:\n"+
```

Comment:
gittar shouldn't create pipeline related link directly.
you should provide a method for invoker to set extra infos, include links.

Reply:
Using the git push command case